synapseclient/models/protocols/table_protocol.py

```diff
@@ -0,0 +1,128 @@
+"""Protocol for the specific methods of this class that have synchronous counterparts
+generated at runtime."""
+
+from typing import Optional, TYPE_CHECKING, Protocol, Union, List
+from typing_extensions import Self
+from synapseclient import Synapse
+from synapseclient.table import (
+    CsvFileTable as Synapse_CsvFileTable,
+    TableQueryResult as Synaspe_TableQueryResult,
+)
+
+
+if TYPE_CHECKING:
+    from synapseclient.models.table import (
+        CsvResultFormat,
+        RowsetResultFormat,
+        Table,
+        Row,
+    )
+
+
+class ColumnSynchronousProtocol(Protocol):
+    """
+    The protocol for methods that are asynchronous but also
+    have a synchronous counterpart that may also be called.
+    """
+
+    def store(self, synapse_client: Optional[Synapse] = None) -> Self:
+        """Persist the column to Synapse.
+
+        :param synapse_client: If not passed in or None this will use the last client
```

Comment:
nit: docstring needs to be updated with Google docs style

Reply:
I will be handling this in https://sagebionetworks.jira.com/browse/SYNPY-1418